clusterctl/clusterdeployer/clusterclient.go

```diff
@@ -129,16 +161,42 @@ func (c *clusterClient) GetMachineObjects() ([]*clusterv1.Machine, error) {
 func (c *clusterClient) CreateClusterObject(cluster *clusterv1.Cluster) error {
 	// TODO: Support specific namespaces https://github.com/kubernetes-sigs/cluster-api/issues/252
 	_, err := c.clientSet.ClusterV1alpha1().Clusters(apiv1.NamespaceDefault).Create(cluster)
+	if err != nil {
+		return fmt.Errorf("error creating cluster: %v", err)
+	}
 	return err
 }
 
+func (c *clusterClient) CreateMachineDeploymentObjects(deployments []*clusterv1.MachineDeployment) error {
+	// TODO: Support specific namespaces https://github.com/kubernetes-sigs/cluster-api/issues/252
+	for _, deploy := range deployments {
+		// TODO: Run in parallel https://github.com/kubernetes-sigs/cluster-api/issues/258
```

Comment:
the issue reference is for machines, do you want to update the issue to include machine deployments and machinesets or file a separate issue?

Reply:
Added a comment on the issue (I can't edit the original text) that MachineSets and MachineDeployments should be included as well.